### ISSUE TITLE
Broken bundlephobia

### ISSUE THREAD

**Issue by frederikhors** [ItalyPaleAle/svelte-spa-router#103]
It never happened to me before: `svelte-spa-router` can't be bundlephobied: https://bundlephobia.com/result?p=svelte-spa-router

Can I ask you why?

**Comment by ItalyPaleAle** [comment]
(Disclaimer: I am not familiar with bundlephobia)

Looking at the error message:

```txt
/tmp/tmp-build/packages/build-svelte-spa-router-lhh/node_modules/svelte-spa-router/Router.svelte 1:0 Module parse failed: Unexpected token (1:0) You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders > <script context="module"> | // Something's wrong with eslint on this file | /* eslint-disable no-multiple-empty-lines */ @ /tmp/tmp-build/packages/build-svelte-spa-router-lhh/index.js 1:10-38
```

Looks like bundlephobia doesn't support Svelte? As you know, to use Svelte with webpack you need to add the svelte-loader and support the `.svelte` extension

**Comment by frederikhors** [comment]
Yea. But I tried other Svelte packages too and no problem! 😄

**Comment by frederikhors** [comment]
Great work with your router's size anyway!

**Comment by ItalyPaleAle** [comment]
I tried other Svelte modules. Some work, some don't.

It seems that those who work are exporting pre-compiled files. This router is not. I have decided not to export a pre-compiled file because:

1. This router wouldn't make sense as a standalone component anyways. It's designed to be used with Svelte 3 and only with it
2. I prefer users to compile the router themselves, so they use the same version of the Svelte compiler as the rest of their code
3. It would add complexity in managing the project and each release, and it would force me to re-release the router just to update the Svelte compiler used, and given points 1 and 2 above it doesn't provide any more value

As for this router's size... Keeping this minimal has always been my main goal, and it's one of the main selling points. I've turned down a lot of feature requests because of that too.

One other important point is that this router only has one runtime dependency, on https://github.com/lukeed/regexparam (which has no runtime dependencies itself). This is to keep the router small, but even more importantly to maintain a lean software supply chain and reduce the risk of security issues. (As a note, starting with version 2.1.0, the exact version of regexparam is also pinned to a precise version so it won't pick up new versions automatically, for security reasons: https://github.com/ItalyPaleAle/svelte-spa-router/releases/tag/v2.1.0)
